sdk/python/setup.py

```diff
@@ -108,6 +108,5 @@ def find_version(*file_path_parts: str) -> str:
         'console_scripts': [
             'dsl-compile = kfp.deprecated.compiler.main:main',
             'dsl-compile-v2 = kfp.compiler.main:main',
-            'kfp=kfp.__main__:main',
```

Comment:
We would need to bring the CLI back soon. Probably keep it as-is, and we fix it shortly?
In the meanwhile, I think we can take a stab to remove `v2` from `dsl-compile-v2`. May need to rename `dsl-compile` to `dsl-compile-v1` or `dsl-compile-deprecated` if it's used in our test infra.